src/app/submission/form/submission-form.component.ts

```diff
@@ -332,4 +349,46 @@ export class SubmissionFormComponent implements OnChanges, OnDestroy {
     );
   }
 
+  /**
+   * Does the shortcut panel should be displayed for this form
```

Comment:
"Indicate if the shortcut panel should be displayed for this form" seems clearer but you could just remove this sentence since the following is explaining in more details.